util/src/zerocopy/mod.rs

```diff
@@ -1,3 +1,13 @@
+//! This module provides utilities for working with zero-copy references and slices.
+//!
+//! It offers the following primary abstractions and traits:
+//!
+//! - [`RefMaker`]: A helper structure for safely creating [`Ref`](zerocopy::Ref) references from byte slices.
+//! - [`ZerocopyEmancipateExt`]: A trait to convert `Ref<B, T>` into a borrowed `Ref<&[u8], T>`.
+//! - [`ZerocopyEmancipateMutExt`]: A trait to convert `Ref<B, T>` into a borrowed mutable `Ref<&mut [u8], T>`.
+//! - [`ZerocopySliceExt`]: Extension methods for parsing byte slices into zero-copy references.
+//! - [`ZerocopyMutSliceExt`]: Extension methods for parsing and zeroizing byte slices into zero-copy references.
```

Comment:
I know this is a bit boring but could we please go for a line length of 80 characters?